src/compiler/performanceCore.ts

```diff
@@ -90,31 +86,6 @@ function tryGetNodePerformanceHooks(): PerformanceHooks | undefined {
             const { performance: nodePerformance, PerformanceObserver } = require("perf_hooks") as typeof import("perf_hooks");
             if (hasRequiredAPI(nodePerformance as unknown as Performance, PerformanceObserver)) {
                 performance = nodePerformance as unknown as Performance;
-                // There is a bug in Node's performance.measure prior to 12.16.3/13.13.0 that does not
-                // match the Web Performance API specification. Node's implementation did not allow
-                // optional `start` and `end` arguments for `performance.measure`.
-                // See https://github.com/nodejs/node/pull/32651 for more information.
-                const version = new Version(process.versions.node);

```

Comment:
Is this still intentionally removed? I don't know that we've guilty Node 12 yet.

Reply:
I think it adds circular dependency, however, I can revert these changes. However, based on public CI, the minimum version of NodeJs is 14.* - or am I wrong? Also, Node.js 12 is outdated.
 
> Node.js 12 will reach End-of-Life status on 30 April 2022, https://nodejs.org/es/blog/release/v12.22.12/#notable-changes

Reply:
Though we do only test back to 14, syntactically and at runtime we're still holding off on dropping even Node 10; but that might be something we change to Node 12 depending on esm stuff.

But, maybe it doesn't matter anyway as these are very old patch versions.

Reply:
But, I will inquire about this when the week starts.

Reply:
Oke